### ISSUE TITLE
Spell Save DC

### ISSUE THREAD

**Issue by sauljaffe** [HarmlessHarm/HarmlessKey#43]
The Spell Save DC idoes not display in the player list.  I don't recall if it displayed before, or if this is just since the other fields (loot, XP, etc.) were added.

**Comment by HarmlessHarm** [comment]
This does work to our knowledge.
It might not work because the field isn't filled in correctly. 
Try turning it off and on again...

**Comment by sauljaffe** [comment]
I logged out and in again and it seems to display correctly.  The reason I re-logged in was because of the other issue I reported with the AC - I’ve changed it several times, re-logged several times, and did a few other things that does not fix it.

> On Jan 10, 2020, at 6:39 PM, Harm Manders <notifications@github.com> wrote:
> 
> This does work to our knowledge.
> It might not work because the field isn't filled in correctly.
> Try turning it off and on again...
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> 

--
Saul Jaffe
sauljaffe@newsguy.com

“Together, we must remove the physical barriers we have created and the social barriers that we have accepted. For ours will never be a truly prosperous nation until all within it prosper.”

President George Bush
upon signing the Americans with Disabilities Act
July 26, 1990
